pkgs/top-level/all-packages.nix

```diff
@@ -32548,7 +32548,7 @@ with pkgs;
 
   nixStatic = pkgsStatic.nix;
 
-  nixops = callPackage ../tools/package-management/nixops { };
+  nixops = builtins.throw "The NixOps 1.x branch is unmaintained and unsupported. Please upgrade to nixopsUnstable and contribute to the NixOps 2 release.";
```

Comment:
This can go in aliases, I think.

Reply:
I don't see why that would be useful. Isn't it nicer to keep the nixops attrs together? Will make the 2.0 release slightly easier.

Reply:
We use aliases to prevent all-packages.nix from being cluttered with 100's of deprecation/removal errors.